firefox-ios/Client/metrics.yaml

```diff
@@ -656,6 +656,67 @@ downloads:
       - fx-ios-data-stewards@mozilla.com
     expires: "2025-01-01"
 
+# Microsurvey
+microsurvey:
```

Comment:
Are we going to have a follow up for the user selection?

Reply:
I was wondering what the value is in capturing the user selection or if capturing the user response is enough, I left a comment on the doc for Ray. Unless you have more details.